HW03/vitaly_kadyrko/vitaly_kadyrko_hw_03_t_01.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class Homework3
+  def task1(log)
+    array_with_lines = log.split("\n")
+    array_with_error_lines = array_with_lines.select { |x| x.downcase.include?('error') }
+    if array_with_error_lines == []
+      ''
+    else
+      array_with_error_lines[0].to_s
+    end
```

Comment:
you can do the same in one line
```suggestion
log.split("\n").find { |x| x.downcase.include?('error') } || ''
```